pyproject.toml

```diff
@@ -105,6 +105,7 @@ typer = {extras = ["all"], version = ">=0.6.1,<0.8.0"}
 types-pkg-resources = "^0.1.3"
 packaging = "<22"
 orjson = "^3.8.3"
+pandas = "^1.5.3"
```

Comment:
Pandas is a heavy dependency we rather not add. Having done a quick look at the code, we mostly use it for reading a csv and extracting values. Please remove the dependency and consider alternative implementations.

Reply:
Done